src/main/java/net/atos/zac/signalering/event/SignaleringEventObserver.java

```diff
@@ -102,6 +102,10 @@ private Signalering buildSignalering(final Signalering signalering, final Signal
                     return addTarget(signalering, subject);
                 }
             }
+            case ZAAK_VERLOPEND, TAAK_VERLOPEN -> {
+                // These are NOT event driven and should not show up here
+                LOG.warning(String.format("ignored SignaleringType %s", event.getObjectType()));
+            }
             default -> LOG.warning(String.format("unknown SignaleringType %s", event.getObjectType()));
```

Comment:
Als je alle enum waarden hebt gebruikt in een case dan is deze default niet nodig want als je iets toevoegd aan de enum en hij zit niet in een case dan krijg je in Java 17 een compile error

Reply:
Pas ik ook aan. Beter at compile time. :-)